website/www/site/content/en/community/contact-us.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Contact Us"
+aliases:
+  - /community/
+  - /use/issue-tracking/
+  - /use/mailing-lists/
+  - /get-started/support/
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Contact Us
+
+There are many ways to reach the Beam user and developer communities - use
+whichever one seems best.
+
+<!-- We use div instead of table shortcode here, because markdownify break superscripts -->
```

Comment:
What is the problem here? What is markdownify and what is superscripts ?

Reply:
If you look at the table shortcode as its mentioned in the comment (its located at website/www/site/layouts/shortcodes/table), you will see we can pass the markdown into a `div` because of `{{ .Inner | markdownify }}`. And we use this table shortcode to replace `{:.table}` syntax in Jekyll. We've done it with several tables. However, only this one, it uses `[^1]` to refer a link and unfortunately `{{ .Inner | markdownify }}` doesn't support this passing, so we need to use the `div` directly on this without passing through `markdownify`.